marine/src/config/marine_config.rs

```diff
@@ -154,12 +155,17 @@ impl MarineModuleConfig {
             }) => {
                 mapped_dirs
                     .values_mut()
-                    .map(|path| *path = root.join(&path))
+                    .map(|path| {
+                        *path = root.join(&path);
+                        log::debug!("mapped_dir after: {}", path.display());
+                    })
                     .for_each(drop);
```

Comment:
Side effects in `map` looks suspicious.  It should go to the `.for_each` instead.